tests/components/lunatone/test_light.py

```diff
@@ -0,0 +1,126 @@
+"""Tests for the Lunatone integration."""
+
+from datetime import timedelta
+from unittest.mock import AsyncMock, PropertyMock, patch
+
+from freezegun.api import FrozenDateTimeFactory
+from lunatone_rest_api_client import Device, Devices
+from lunatone_rest_api_client.models import ControlData
+from syrupy.assertion import SnapshotAssertion
+
+from homeassistant.components.light import DOMAIN as LIGHT_DOMAIN
+from homeassistant.const import (
+    ATTR_ENTITY_ID,
+    SERVICE_TURN_OFF,
+    SERVICE_TURN_ON,
+    STATE_OFF,
+    STATE_ON,
+    Platform,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import device_registry as dr, entity_registry as er
+
+from . import setup_integration
+
+from tests.common import MockConfigEntry, async_fire_time_changed
+
+TEST_ENTITY_ID = "light.device_1"
+
+
+async def test_setup(
+    hass: HomeAssistant,
+    mock_lunatone_devices: AsyncMock,
+    mock_lunatone_info: AsyncMock,
+    mock_config_entry: MockConfigEntry,
+    device_registry: dr.DeviceRegistry,
+    entity_registry: er.EntityRegistry,
+    snapshot: SnapshotAssertion,
+) -> None:
+    """Test the Lunatone configuration entry loading/unloading."""
+    await setup_integration(hass, mock_config_entry)
+
+    entities = hass.states.async_all(Platform.LIGHT)
+    for entity_state in entities:
+        entity_entry = entity_registry.async_get(entity_state.entity_id)
+        assert entity_entry == snapshot(name=f"{entity_entry.entity_id}-entry")
+        assert entity_state == snapshot(name=f"{entity_entry.entity_id}-state")
+
+
+async def test_turn_on_off(
+    hass: HomeAssistant,
+    mock_lunatone_devices: AsyncMock,
+    mock_lunatone_info: AsyncMock,
+    mock_config_entry: MockConfigEntry,
+) -> None:
+    """Test the light can be turned on and off."""
+
+    def fake_control(device: Device, control_data: ControlData):
+        if control_data.switchable is not None:
+            device._data.features.switchable.status = control_data.switchable
+
+    device_list = [
+        Device(mock_lunatone_devices._auth, d)
+        for d in mock_lunatone_devices._data.devices
+    ]
```

Comment:
lets mock the `Device` list in `mock_lunatone_devices` since it is part of having a complete mock, right?